packages/components/src/components/primitives/CloseButton/types.ts

```diff
@@ -0,0 +1,10 @@
+import { IconButtonProps as ChakraIconButtonProps } from '@chakra-ui/react';
+
+export const CloseButtonSizes = ['s', 'm', 'l'] as const;
+
+export type CloseButtonSize = typeof CloseButtonSizes[number];
+
+export interface CloseButtonProps extends Omit<ChakraIconButtonProps, 'aria-label'> {
+  size?: CloseButtonSize;
+  "aria-label"?: string;
```

Comment:
why are you omitting the aria-label to add it again?

Reply:
in this case because we are using the `IconButton` and for this component that prop is required, what I did was to set a default `aria-label` as `close button` and set it as an optional prop if the user wants to change it.